components/application-connectivity-validator/internal/validationproxy/handler_test.go

```diff
@@ -30,6 +36,36 @@ type event struct {
 	Title string `json:"title"`
 }
 
+var (
+	applicationManagedByCompass = &v1alpha12.Application{
+		TypeMeta: metav1.TypeMeta{
+			Kind:       "Application",
+			APIVersion: "applicationconnector.kyma-project.io/v1alpha1",
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			Name: applicationMetaName,
+		},
+		Spec: v1alpha12.ApplicationSpec{
+			Description:     "Description",
+			Services:        []v1alpha12.Service{},
+			CompassMetadata: &v1alpha12.CompassMetadata{Authentication: v1alpha12.Authentication{ClientIds: []string{applicationName}}},
```

Comment:
We should not use `applicationName` as a `ClientIds` as it caused our test to pass despite the logic error.

Reply:
Done!